guides/introduction/Embedded Schemas.md

```diff
@@ -62,9 +62,15 @@ defmodule User do
 end
 ```
 
+Defining embedded schema in such a way is called inline-defined embedded schemas which are:
+- the simpliest way to define embedded schema
```

Comment:
```suggestion
```
I would personally take this out, given the other use case that is highlighted further down:

> Like was said before, they are persistence agnostic. This makes embedded schemas ideal for scenarios where you want to manage structured data without necessarily persisting it.

So it's only the simplest if you aren't doing the above.

Reply:
That's reasonable point of view. Besides this I have a typo in this line, so I'll totally remove it, like you suggested.
Thank you.